playbooks/reboot-hosts.yml

```diff
@@ -4,8 +4,14 @@
   gather_facts: no
   tasks:
 
+  - name: check whether consul is installed or not
+    stat:
+      path: /bin/consul
```

Comment:
Seems like it's a wrong location of consul binary. Could you please double check it?